scripts/storage/secondary/createtmplt.sh

```diff
@@ -131,92 +123,66 @@ do
   f)	fflag=1
 		tmpltimg="$OPTARG"
 		;;
-  s)	sflag=1
-		;;
-  d)	dflag=1
-		descr="$OPTARG"
-		;;
-  S)	Sflag=1
-		size=$OPTARG
-                let "size>>=10"
-		ulimit -f $size
-		;;
-  h)	hflag=1
-		hvm="true"
-		;;
-  u)	cleanup="true"
-		;;
-  v)	verbose="true"
-		;;
```

Comment:
@svenvogel @DennisKonrad several flags/opts are removed from the script, do you think it will cause regression in case the script is evoked via mgmt server or ssvm agent?

Reply:
@rhtyd yes maybe. can you see how/where the createtmplt.sh will called? can you see wich options are important?

Reply:
@rhtyd if i could know which are important for check i can put them back.

Reply:
Let me review the results, do code investigation and get back to you tomorrow.

Reply:
@rhtyd thank you for help. you investigation will help to find a good solution.

Reply:
@rhtyd could you find something?

Reply:
@svenvogel cc @ustcweizhou  - can you put back all the flags back? The issue is likely that your changes are breaking usage of this script. You can use `git grep <script name>` and use an IDE to find usages of the script.